### PR TITLE
Slack

### PR DESCRIPTION
**Pull request by ryankbales**
Adds slack gem to notify ping-pong room of match results.

### CODE REVIEW

**Comment by ryankbales** [comment]
We will need to figure out where to put the Slack token constant on the server.

**Comment by colto** [comment]
:shipit:

File: app/controllers/matches_controller.rb [hunk 1]
```diff
@@ -74,6 +75,11 @@ def create_new_match
     @game_2.update_attributes(match_id: @match.id) if @game_2
     @game_3.update_attributes(match_id: @match.id) if @game_3
     @match.update_player_rankings
+    winner_name = "#{@match.winner[0].first_name}" + " #{@match.winner[0].last_name} "
+    loser_name = "#{@match.loser[0].first_name}" + " #{@match.loser[0].last_name} "
+    match_message = winner_name + " has defeated " + loser_name + "#{@match.winner[0].game_wins} games to " + "#{@match.loser[0].game_wins}!"
+    slack = Slack::Notifier.new "https://hooks.slack.com/services/#{ENV["SLACK_TOKEN"]}", channel: '#g5_pingpong', username: 'DeployTron5000'
```
Comment (colto) [inline app/controllers/matches_controller.rb]: Looks good. You can set env variables in Heroku pretty easily. You just have to have write access to the Heroku app. 

`heroku config:set SLACK_TOKEN=123abc456ryansucks789`

And might suggest we change the name from DeployTron to PingBot or something more fitting?

**Comment by variousred** [comment]
awesome :shipit:

**Comment by ryankbales** [comment]
So I'll merge as soon as @thescubageek gives me his blessing.  Plus, he's the only one that can deploy and add the SLACK token to Heroku env variables.

**Comment by thescubageek** [comment]
@ryankbales Looks good, you have my blessing to :ship: . Email me the slack ENV var and I'll set it up on Heroku.

**Pull request merged**